cvat/apps/iam/rules/server.rego

```diff
@@ -30,10 +30,15 @@ allow {
 }
 
 allow {
-    input.scope == utils.SEND_LOGS
+    input.scope == utils.SEND_EVENTS
 }
 
 allow {
     input.scope == utils.LIST_CONTENT
     utils.has_perm(utils.WORKER)
 }
+
+allow {
+    input.scope == utils.LIST_EVENTS
+    utils.has_perm(utils.BUSINESS)
```

Comment:
Why business?

Which code is responsible for filtration? Can I get all events? IAM should decide which events I can get. For example, if I a maintainer of an organization, I can see all events of the organization. At the same time, if I'm a worker, I can see only my own events.

Reply:
Implemented by @kirill-sizov